### ISSUE TITLE
(committee list API): returns 500 "Reverse for 'committee_details' with keyword arguments not found"

### ISSUE THREAD

**Issue by ryparker** [LibraryOfCongress/api.congress.gov#162]
Requesting committee list returns a 500 error

Request
```sh
curl 'https://api.congress.gov/v3/committee?format=json&limit=250&api_key=<REPLACE_ME>'
```

Response
```json
{
    "error": "Reverse for 'committee_details' with keyword arguments '{'systemCode': 'LOC18600316', 'chamber': 'house'}' not found. 2 pattern(s) tried: ['v3/committee/(?P<chamber>(house|senate|joint|House|Senate|Joint))/(?P<systemCode>([a-z0-9]+))(?P<file_ext>(\\\\.xml|\\\\.json|/))?$', 'v2/committee/(?P<chamber>(house|senate|joint|House|Senate|Joint))/(?P<systemCode>([a-z0-9]+))(?P<file_ext>(\\\\.xml|\\\\.json|/))?$'] (NoReverseMatch)",
    "request": {
        "contentType": "application/json",
        "format": "json"
    }
}
```

**Comment by jonquandt** [comment]
We're also seeing this issue on our end during our weekly checks.

**Comment by jonquandt** [comment]
Just an update that now we're seeing that the issue has changed. We're now seeing a list come back from the basic
https://api.congress.gov/v3/committee?format=json&api_key=DEMO_KEY request, but the url elements appear to be using a systemCode identifier - e.g. https://api.congress.gov/v3/committee/senate/LOC18261226?format=xml, which has an LOCXXXXXX identifier that leads to a different error:

```json
{
    "error": "'NoneType' object has no attribute 'in_current_congress' (AttributeError)",
    "request": {
        "chamber": "senate",
        "contentType": "application/json",
        "format": "json",
        "systemCode": "loc18200208"
    }
}
```

I may have missed it, but I don't see anything in the [changelog](https://github.com/LibraryOfCongress/api.congress.gov/blob/main/ChangeLog.md) related to this endpoint and issue. Given the change of Congress session, we are anticipating that there may be a larger number of updates to committees in the next month or so.

**Comment by mqueen73** [comment]
Jon,

At the request of Stephen Kharfen (GPO), I have started to create historical committee authority records for terminated committees so that GPO can access them via the Congress.gov API. The historical committees predate the semi-standardized House and Senate-issued committee codes. The system codes we are piloting are the terminated committee’s start year (“1879”), month (“05”), and day (“16”) prefixed with “LOC.”

Sorry, we should have pre-coordinated with your team about these codes. Can they work for you?

As a short term fix, do you need us to remove them until you can program for them?

With the exception of the external committee code values (e.g. LOC18790516), each committee authority record is created in Congress.gov Admin Tools and formatted/organized in the same manner as current committee authority records.

**Comment by jonquandt** [comment]
Chatting with the team to see about the LOC system codes, but I don't anticipate an issue at this time as long as those remain stable.

**Comment by apreiter18** [comment]
> Just an update that now we're seeing that the issue has changed. We're now seeing a list come back from the basic https://api.congress.gov/v3/committee?format=json&api_key=DEMO_KEY request, but the url elements appear to be using a systemCode identifier - e.g. https://api.congress.gov/v3/committee/senate/LOC18261226?format=json, which has an LOCXXXXXX identifier that leads to a different error:
> 
> ```json
> {
>     "error": "'NoneType' object has no attribute 'in_current_congress' (AttributeError)",
>     "request": {
>         "chamber": "senate",
>         "contentType": "application/json",
>         "format": "json",
>         "systemCode": "loc18261226"
>     }
> }
> ```
> 
> I may have missed it, but I don't see anything in the [changelog](https://github.com/LibraryOfCongress/api.congress.gov/blob/main/ChangeLog.md) related to this endpoint and issue. Given the change of Congress session, we are anticipating that there may be a larger number of updates to committees in the next month or so.
> 
> @apreiter18 do you have any information on ETA for fixes for this issue (and #180)?

@jonquandt - we are estimating that the fix will be available on February 5, 2024.
